localstack/aws/spec-patches.json

```diff
@@ -115,6 +115,146 @@
       "op": "add",
       "path": "/shapes/GetBucketLocationOutput/payload",
       "value": "LocationConstraint"
+    },
+    {
+      "op": "add",
+      "path": "/shapes/BucketAlreadyOwnedByYou/members/BucketName",
+      "value": {
+        "shape": "BucketName"
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/GetObjectOutput/members/StatusCode",
+      "value": {
+        "shape": "GetObjectResponseStatusCode",
+        "location": "statusCode"
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/NoSuchKey/members/Key",
+      "value": {
+        "shape": "ObjectKey"
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/NoSuchKey/error",
+      "value": {
+        "httpStatusCode": 404
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/PreconditionFailed",
+      "value": {
+        "type": "structure",
+        "members": {
+          "Condition": {
+            "shape": "IfCondition"
+          }
+        },
+        "error": {
+          "httpStatusCode": 412
+        },
+        "documentation": "<p>At least one of the pre-conditions you specified did not hold</p>",
+        "exception": true
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/IfCondition",
+      "value": {
+        "type": "string"
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/InvalidRange",
+      "value": {
+        "type": "structure",
+        "members": {
+          "ActualObjectSize": {
+            "shape": "ObjectSize"
+          },
+          "RangeRequested": {
+            "shape": "ContentRange"
+          }
+        },
+        "error": {
+          "httpStatusCode": 416
+        },
+        "documentation": "<p>The requested range is not satisfiable</p>",
+        "exception": true
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/HeadObjectOutput/members/Expires",
+      "value": {
+        "shape": "Expires",
+        "documentation": "<p>The date and time at which the object is no longer cacheable.</p>",
+        "location": "header",
+        "locationName": "expires"
+      }
+    },
+    {
+      "op": "add",
+      "path": "/shapes/GetObjectOutput/members/Expires",
+      "value": {
+        "shape": "Expires",
+        "documentation": "<p>The date and time at which the object is no longer cacheable.</p>",
+        "location": "header",
+        "locationName": "expires"
+      }
+    },
+    {
+      "op": "remove",
+      "path": "/shapes/DeleteObjectsOutput"
+    },
+    {
+      "op": "add",
+      "path": "/shapes/DeleteResult",
```

Comment:
If I remember well, it's because the response is wrapped in the name of the output shape. Like <DeleteObjectOutput>response members xml</DeleteObjectOutput>.
But S3 returns <DeleteResult> instead. Not sure if I can change the name in any other way. Doesn't seem to have `locationName` for output shapes. Seems weird and dirty though 🤷‍♂️